homework/style.css

```diff
@@ -1,3 +1,56 @@
+.root {
```

Comment:
On a wide screen it would better to set a maximum width of your items.

![image](https://user-images.githubusercontent.com/2788771/65252936-16295f00-dafa-11e9-9c02-7605a42778f6.png)